container/container.go

```diff
@@ -1,201 +1,523 @@
+//
+// See also Name.
+func SetName(cnr *Container, name string) {
+	cnr.SetAttribute(attributeName, name)
 }
 
-func (c *Container) BasicACL() (res acl.Basic) {
-	res.FromBits(c.v2.GetBasicACL())
-	return
+// Name returns container name set using SetName.
+//
+// Zero Container has no name.
+func Name(cnr Container) string {
+	return cnr.Attribute(attributeName)
 }
 
-func (c *Container) SetBasicACL(v acl.Basic) {
-	c.v2.SetBasicACL(v.Bits())
+// SetCreationTime writes container's creation time in Unix Timestamp format.
+//
+// See also CreatedAt.
+func SetCreationTime(cnr *Container, t time.Time) {
+	cnr.SetAttribute(attributeTimestamp, strconv.FormatInt(t.Unix(), 10))
 }
 
-func (c *Container) Attributes() Attributes {
-	return NewAttributesFromV2(c.v2.GetAttributes())
+// CreatedAt returns container's creation time set using SetCreationTime.
+//
+// Zero Container has zero timestamp (in seconds).
+func CreatedAt(cnr Container) time.Time {
+	var sec int64
+
+	attr := cnr.Attribute(attributeTimestamp)
+	if attr != "" {
+		var err error
+
+		sec, err = strconv.ParseInt(cnr.Attribute(attributeTimestamp), 10, 64)
+		if err != nil {
+			panic(fmt.Sprintf("parse container timestamp: %v", err))
+		}
+	}
+
+	return time.Unix(sec, 0)
 }
 
-func (c *Container) SetAttributes(v Attributes) {
-	c.v2.SetAttributes(v.ToV2())
+// SetSubnet places the Container on the specified NeoFS subnet. If called,
+// container nodes will only be selected from the given subnet, otherwise from
+// the entire network.
+func SetSubnet(cnr *Container, subNet subnetid.ID) {
+	cnr.SetAttribute(container.SysAttributeSubnet, subNet.EncodeToString())
 }
 
-func (c *Container) PlacementPolicy() *netmap.PlacementPolicy {
-	m := c.v2.GetPlacementPolicy()
-	if m == nil {
-		return nil
+// Subnet return container subnet set using SetSubnet.
+//
+// Zero Container is bound to zero subnet.
+func Subnet(cnr Container) (res subnetid.ID) {
+	val := cnr.Attribute(container.SysAttributeSubnet)
+	if val != "" {
+		err := res.DecodeString(val)
+		if err != nil {
+			panic(fmt.Sprintf("invalid subnet attribute: %s (%v)", val, err))
+		}
 	}
 
-	var p netmap.PlacementPolicy
-	// FIXME(@cthulhu-rider): #225 handle error
-	err := p.ReadFromV2(*m)
-	if err != nil {
-		panic(err)
+	return
+}
+
+const (
+	attributeHomoHashKey     = container.SysAttributePrefix + "DISABLE_HOMOMORPHIC_HASHING"
+	attributeHomoHashEnabled = "true"
+)
+
+// DisableHomomorphicHashing sets flag to disable homomorphic hashing of the
+// Container data.
+//
+// See also IsHomomorphicHashingDisabled.
+func DisableHomomorphicHashing(cnr *Container) {
+	cnr.SetAttribute(attributeHomoHashKey, attributeHomoHashEnabled)
+}
+
+// IsHomomorphicHashingDisabled checks if DisableHomomorphicHashing was called.
+//
+// Zero Container has enabled hashing.
+func IsHomomorphicHashingDisabled(cnr Container) bool {
+	return cnr.Attribute(attributeHomoHashKey) == attributeHomoHashEnabled
+}
```

Comment:
Why did you put it here instead of waiting for api-go PR?

Reply:
Async with @carpawell, I'll wait for the PR in api-go.